packages/react-components/src/testing/alarms/mockProperties.ts

```diff
@@ -0,0 +1,103 @@
+import {
+  AssetModelProperty,
+  AssetProperty,
+  AssetPropertyValue,
+} from '@aws-sdk/client-iotsitewise';
+import {
+  ALARM_SOURCE_PROPERTY_NAME,
+  ALARM_STATE_PROPERTY_NAME,
+  ALARM_TYPE_PROPERTY_NAME,
+} from '../../hooks/useAlarms/constants';
+import { MOCK_ALARM_INPUT_PROPERTY_ID } from './mockIds';
+
+export const mockStateAssetProperty: AssetProperty = {
```

Comment:
Nit: I would use `satisfies` instead of a type annotation when possible for test data.